packages/UI/examples/html-tag-example/index.html

```diff
@@ -7,17 +7,20 @@
     <title>Threefold UI</title>
 
     <!-- Include the PDF style component, uncomment the script line after copying the build js/css in the same dir -->
-    <!-- <link rel="stylesheet" href="./style.css"> -->
+    <link rel="stylesheet" href="./tailwind.css">
```

Comment:
What do you think if we added some dummy files in the same dir, the `tailwind.css` and  `threefold-ui.umd.js` with no content and one single comment in each file to tell the user that he just needs to copy the build and past them here in these files?

Reply:
This has been already explained in the docs [here](https://github.com/threefoldtech/tfgrid-sdk-ts/blob/development/packages/UI/docs/script_editor.md#building-the-script-editor).

Reply:
Well done